src/contracts/resource.js

```diff
@@ -1,13 +1,14 @@
 // Resource contract
 
 class Resource {
-  constructor(vehicleType, plateNumber, identificationNumber, crew) {
+  constructor(vehicleType, plateNumber, identificationNumber, captain, crew, status, type) {
```

Comment:
Maybe at some point we could extract some objects because it has already 7 parameters.